src/main/java/com/splunk/hecclient/Indexer.java

```diff
@@ -192,6 +188,15 @@ public CloseableHttpResponse run() {
         return readAndCloseResponse(resp);
     }
 
+    private void getSubject() throws LoginException {
```

Comment:
This should return the Subject object and assign it to serviceSubject in constructor.